pkg/disttask/framework/storage/task_table.go

```diff
@@ -21,16 +21,16 @@ import (
 	"sync/atomic"
 
 	"github.com/docker/go-units"
-	"github.com/ngaut/pools"
 	"github.com/pingcap/errors"
 	"github.com/pingcap/failpoint"
 	"github.com/pingcap/tidb/pkg/disttask/framework/proto"
 	"github.com/pingcap/tidb/pkg/kv"
 	"github.com/pingcap/tidb/pkg/sessionctx"
 	"github.com/pingcap/tidb/pkg/sessionctx/variable"
+	"github.com/pingcap/tidb/pkg/util"
 	"github.com/pingcap/tidb/pkg/util/chunk"
 	"github.com/pingcap/tidb/pkg/util/sqlexec"
-	"github.com/tikv/client-go/v2/util"
+	cliutil "github.com/tikv/client-go/v2/util"
```

Comment:
`cliutil` seems quite confusing, because `CLI` usually represents `Command Line Interface`. Maybe we should use the full name or use `clit`.

Reply:
Changed to `clit`.